mockwebserver-junit5/src/main/kotlin/mockwebserver3/junit5/internal/MockWebServerInstance.kt

```diff
@@ -0,0 +1,3 @@
+package mockwebserver3.junit5.internal
+
+annotation class MockWebServerInstance(val tls: Boolean)
```

Comment:
Remove parameter, it’s too arbitrary? Programmatic configuration is easy enough, and where it’s not we should fix that.